features/apps/src/main/feature/feature.xml

```diff
@@ -43,6 +43,7 @@
         <feature>error-api</feature>
         <feature>error-impl</feature>
         <feature>util-uuidgenerator</feature>
+        <feature>platform-solr-app</feature>
```

Comment:
:exclamation: Please move this feature to `platform-app`.  The persistence store should not be required by the core catalog framework.  You might have to add `solr-core` to `platform-core` though.